protocol/lib/error/logging.go

```diff
@@ -36,3 +40,14 @@ func WrapErrorWithSourceModuleContext(err error, module string) error {
 	return NewErrorWithLogContext(err).
 		WithLogKeyValue(SourceModuleKey, fmt.Sprintf("x/%v", module))
 }
+
+// LogErrorWithBlockHeight logs an error using the cosmos sdk Context's logger, appending the block
+// height to the error message.
+func LogErrorWithBlockHeight(ctx sdk.Context, err error) {
```

Comment:
Any way to test this function?

Specifically a test that verifies it doesn't panic when provided a nil `err` or something